### ISSUE TITLE
Disable plating interface until at least one study is selected

### ISSUE THREAD

**Issue by fedarko** [biocore/LabControl#577]
Based on discussion with @charles-cowart.

**Comment by AmandaBirmingham** [comment]
I agree that 99% of the time (ok, probably 99.9% of the time ...) users will want to plate actual samples.  However, it is entirely possible for them to make and run plates that contain nothing but some combination of blanks, negative controls, and positive controls--none of which require a study.  Therefore I do not think we can require them to pick a study before letting them plate.  We could, maybe, ask them if they really want to continue without picking a study--that would probably be an acceptable level of nagging, given how infrequently they will probably legitimately want to do a no-study plate.  Thoughts?

**Comment by fedarko** [comment]
I wasn't aware that was a use case, but I can understand how that'd happen! Agreed that we should definitely allow users to plate without a study, then.

As I remember, the main reason Charlie and I started talking about this issue was #568—currently, if you start plating things then select a study, the already-plated wells don't update and it looks wonky. If we make sure that #568 is properly accounted for, I'm ok with not going ahead with this (or at least delegating to the asking-for-confirmation-before-plating-without-a-study idea).
